### PR TITLE
feat: [LIB-4] add question accessiblity templates

### PR DESCRIPTION
**Pull request by VaradP27**
## Does your PR contain necessary tests?

All patches which change the editor code must include tests. You can always read more
on [PR testing](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_contributing_code.html#tests),
[how to set the testing environment](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html) and
[how to create tests](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [x] Manual tests

## What changes did you make?

*Give an overview…*

### CODE REVIEW

**Comment by Ajitesh13** [comment]
@VaradP27 please update the verison in package.json and add that corresponding version to the change log.

File: plugins/templates/templates/default.js [hunk 1]
```diff
@@ -15,13 +15,13 @@ CKEDITOR.addTemplates( 'default', {
...
...d be expanded when the question opens. Open Sample 0, close the other sections and press <source> at the top of this window to automatically add open="" to the details tag for this section. While in the source view, edit the style to open="open". You can manually insert the tag in the source window too.</p><p>Explanatory text and diagrams (svg). Like the Example section, keep the text to a minimum. Note that the Sample Input and Output sections show input and expected results clearly. They generally do not need to be repeated here.</p></div></details><details><summary class="section-title" aria-level="4" role="heading">Sample Case 1</summary><div class="collapsable-details"><p aria-level="5" role="heading" class=”section-title”><strong>Sample Input 1</strong></p><pre> STDIN Function ----- -------- < as an example, a single element array called arr is the input> 1 the size of arr[] n = 1 2 arr = [2] </pre><p aria-level="5" role="heading" class=”section-title”><strong>Sample Output 1</strong></p><pre> output exactly as it is expected </pre><p aria-level="5" role="heading" class=”section-title”><strong>Explanation</strong></p><p>Explanatory text and diagrams</p></div></details></div>'
```
Comment (SatyamSetia) [inline plugins/templates/templates/default.js:18]: Shouldn't we wrap the content in html and body tags ?

Comment (VaradP27) [reply plugins/templates/templates/default.js]: Done, made the required changes

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ﻿CKEditor 4 Changelog
 ====================
 
+## CKEditor 4.11.4.12
+- [#15](https://github.com/interviewstreet/ckeditor-dev/pull/16)
```
Comment (SatyamSetia) [inline CHANGES.md:5]:
```suggestion
- [#16](https://github.com/interviewstreet/ckeditor-dev/pull/16)
```

**Review by Ajitesh13 — APPROVED** [review]

**Pull request closed without merging**